app/src/components/market/sections/market_sell/market_sell.tsx

```diff
@@ -144,10 +145,12 @@ const MarketSellWrapper: React.FC<Props> = (props: Props) => {
       const holdingsOfOtherOutcomes = holdings.filter((item, index) => {
         return index !== outcomeIndex
       })
-
+      const roundingFactor = STANDARD_DECIMALS - 4
```

Comment:
It looks like `amountShares` isn't actually always 18 decimals, but instead tracks the amount of decimals of the collateral